src/Stratis.Features.FederatedPeg/SourceChain/DepositExtractor.cs

```diff
@@ -121,7 +121,7 @@ private void ProcessInterFluxBurnRequests(List<IDeposit> deposits, int inspectFo
             {
                 if (inspectForDepositsAtHeight == burnRequest.BlockHeight)
                 {
-                    this.logger.LogInformation($"Processing burn request '{burnRequest.RequestId}' to '{burnRequest.DestinationAddress}' for {new Money(burnRequest.Amount)} STRAX at height {inspectForDepositsAtHeight}.");
+                    this.logger.LogInformation($"Processing burn request '{burnRequest.RequestId}' to '{burnRequest.DestinationAddress}' for {burnRequest.Amount} STRAX at height {inspectForDepositsAtHeight}.");
```

Comment:
Perhaps we need "fix" this here as we rely quite a bit on the logs to debug/test?